anchore_engine/services/catalog/api/controllers/objects.py

```diff
@@ -94,3 +94,61 @@ def delete_object(bucket, archiveid):
         )
 
     return return_object, httpcode
+
+
+@flask_metrics.do_not_track()
+@authorizer.requires_account(with_types=INTERNAL_SERVICE_ALLOWED)
+def create_raw_object(bucket, archiveid, bodycontent):
+    httpcode = 500
+    try:
+        account_name = ApiRequestContextProxy.namespace()
+        obj_mgr = anchore_engine.subsys.object_store.manager.get_manager()
+
+        obj_mgr.put(account_name, bucket, archiveid, bodycontent)
+
+        my_svc = ApiRequestContextProxy.get_service()
+        if my_svc is not None:
+            resource_url = (
+                my_svc.service_record["base_url"]
+                + "/"
+                + my_svc.service_record["version"]
+                + "/archive/"
+                + bucket
+                + "/"
+                + archiveid
```

Comment:
this looks like a good candidate for `os.path.join`, any reason why that wouldn't be useful here?

Reply:
hmmm doesn't os.path.join change behavior on windows? i know that we aren't cross compatible, but maybe it would be more prudent to use "/".join() instead here.

Reply:
updated with "/".join()